solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -202,15 +209,7 @@ library DKG {
         );
         require(!hasDkgTimedOut(self), "dkg timeout already passed");
 
-        verify(
-            self,
-            result.submitterMemberIndex,
-            result.groupPubKey,
-            result.misbehavedMembersIndices,
-            result.signatures,
-            result.signingMembersIndices,
-            result.members
-        );
+        uint32[] memory signingMembers = verify(self, result);
```

Comment:
@nkuba Was there any particular reason it was done this way? Gas costs maybe?

Reply:
Can't compare gas usage as the original version errors out with `Stack too deep` at compilation stage 🤷

Reply:
Problem solved then.